frontend/components/panels/VaultLiquidationLimitsCheckPanel.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+    <BasePanel :current-state="currentStateAndTitle.name">
+        <template #title>{{ currentStateAndTitle.title }}</template>
+        <TextBlock v-if="isExplanationsShown"></TextBlock>
+        <div class="flex justify-between mt-4">
+            <span>Current global limit</span>
+            <span v-if="liquidationLimits"
+                ><Explain :text="liquidationLimits.maximumProtocolDebtDai.toFixed()"></Explain> -
+                <Explain :text="liquidationLimits.currentProtocolDebtDai.toFixed()"></Explain> -
+                <Explain :text="debtDai.toFixed()"></Explain> =
+                <FormatCurrency :value="globalDifference" currency="DAI"
+            /></span>
+            <div v-else>
+                <span class="opacity-50">Unknown</span>
+                <span>DAI</span>
+            </div>
+        </div>
+        <div class="flex justify-between">
+            <span>Current {{ collateralType }} limit</span>
+            <span v-if="liquidationLimits"
+                ><Explain :text="liquidationLimits.maximumCollateralDebtDai.toFixed()"></Explain> -
+                <Explain :text="liquidationLimits.currentCollateralDebtDai.toFixed()"></Explain> -
+                <Explain :text="debtDai.toFixed()"></Explain> =
+                <FormatCurrency :value="collateralDifference" currency="DAI"
+            /></span>
+            <div v-else>
+                <span class="opacity-50">Unknown</span>
+                <span>DAI</span>
+            </div>
+        </div>
+        <div class="flex justify-end mt-4">
+            <BaseButton
+                type="primary"
+                :disabled="isRefreshing"
+                :is-loading="isRefreshing"
+                @click="$emit('refreshLimits')"
+            >
+                Refresh
+            </BaseButton>
+        </div>
+    </BasePanel>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import BigNumber from 'bignumber.js';
+import { LiquidationLimits } from 'auctions-core/src/types';
+import BaseButton from '~/components/common/inputs/BaseButton.vue';
+import TextBlock from '~/components/common/other/TextBlock.vue';
+import Explain from '~/components/common/other/Explain.vue';
+import FormatCurrency from '~/components/common/formatters/FormatCurrency.vue';
+import BasePanel from '~/components/common/other/BasePanel.vue';
+
+export default Vue.extend({
+    name: 'VaultLiquidationLimitsCheckPanel',
+    components: {
+        BaseButton,
+        BasePanel,
+        TextBlock,
+        Explain,
+        FormatCurrency,
+    },
+    props: {
+        liquidationLimits: {
+            type: Object as Vue.PropType<LiquidationLimits>,
+            default: undefined,
```

Comment:
Please see the edit https://github.com/sidestream-tech/unified-auctions-ui/pull/461#discussion_r968408806

Actually we expect to receive the whole `VaultTransaction` in the transaction page, so it wouldn't be possible to receive empty `liquidationLimits`, but empty keys of the object